torch/onnx/symbolic_opset9.py

```diff
@@ -2240,9 +2239,46 @@ def _std(g, input, dim, unbiased, keepdim):
 # torch.std(input, dim, keepdim=False, unbiased=True)
 def std(g, input, *args):
     if len(args) == 3:
-        return _std(g, input, *args)
+        var, _ = _var(g, input, *args)
+    else:
+        var, _ = _var(g, input, None, args[0], None)
+    return g.op("Sqrt", var)
+
+
+# Since position of optional arguments can change for std, this is a hack to find if first argument
+# is 'dim' or 'unbiased'. As shown below, 'dim' argument could be listed before 'unbiased' :
+# torch.var(input, unbiased=True)
+# torch.var(input, dim, keepdim=False, unbiased=True)
+def var(g, input, *args):
+    if len(args) == 3:
+        var, _ = _var(g, input, *args)
+    else:
+        var, _ = _var(g, input, None, args[0], None)
+    return var
+
+
+# Since position of optional arguments can change for std, this is a hack to find if first argument
```

Comment:
```suggestion
# Since position of optional arguments can change for var_mean, this is a hack to find if first argument
```
likewise for the other comments.

Question for the optional values in the comment, `... keepdim=False, unbiased=True ...` They are only optional at PyTorch code level correct? Are we guaranteed to receive either among
```
# torch.var_mean(input, unbiased)
# torch.var_mean(input, dim, keepdim, unbiased)
```

Otherwise it is possible someone actually calls with `torch.var_mean(input, dim)`, and `dim` got considered as `unbiased`

Reply:
Yeah these are the two schemas. I'll fix the comments to be more clear. Thanks.